addons/packages/kapp-controller/0.30.0/bundle/config/values.yaml

```diff
@@ -7,8 +7,8 @@ kappController:
   createNamespace: true
   globalNamespace: tanzu-package-repo-global
   deployment:
+    coreDNSIP: 100.64.0.10
```

Comment:
Lets not change the default behavior and potentially create unintended side effects for TCE, make the default empty string.